forem/forem
# Create optimized_image_tag helper

## What type of PR is this? (check all applicable)

## Description
Created a helper method to replace the use of `cl_image_tag`.

Shoutout to @citizen428 's for his suggestion.
## Related Tickets & Documents
n/a
## QA Instructions, Screenshots, Recordings
Browse around locally (including podcast page) to make sure image are loading properly.

## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
n/a
